package.json

```diff
@@ -278,7 +278,7 @@
     "shellcheck": "^1.1.0",
     "style-loader": "^2.0.0",
     "time-analytics-webpack-plugin": "^0.1.17",
-    "type-fest": "^3.12.0",
+    "type-fest": "^4.10.2",
```

Comment:
bumped this to fix [this example](https://www.typescriptlang.org/play?#code/KYDwDg9gTgLgBDAnmYcDyBbAljAkgOwBNQBlLAc3wEMYBXKYAHjQCMArYAYxgBVlgAfHAC8cAN4AoAJABtANLBEfFHCz44Aa0UQAZunZde-OFQDO4gL5xQMYEXMAlLtEKMFS-gBo4tfBvwQAO74AtJSAPxw+MAAbsBQYQBccO7KwAC6yawc3GnyimnpANwSFiUSoJCwCMYAClicGgTEIGSUNPRM2YZpQqKSsqnGapraet25xmaW1iC29nBOnC5uBV4+fgHBoVIRKWsoSVGx8Zn6OUYo+R4oxaXlldDwSCpkGGAANlg6iIw8feJrmlVOotIhdHAeGceED+OkrAAySzlF6oN6fYAAWXi5CYABFgKYYGoaFgIPhvCQIPROIIROJpNcQaNwXoCUSScTySZzO5ZvNCOYwRCqTTUJFonEoHBku4zuzidQufhrndEXBRVBaQ9wE8aipsVBcYwFZyyRSNdStXThBJ0V8fow7Vh3h8sTimPVGs1SBQlZ0TYTFaTyQJvF6mkRfe06AxGJraQJQkj7e6jV1sHgo60-R046aleaw+hMz6czGAwnBKEBOUAPR1jU8ACCDh4syoruAEgkDchAAssOZAtANOYdGpu6i4JjEIbcYQAHLQDBUD70+dMMT94AfD4QZJEqBqcgWbxicgQCCEFiIYCHmDH-Cn2sSHS+bjmuAYRDLqCrj4ADEPwACm3Xd928S9r1vYALGSWdNyXFc1wASgZOA4GWfBTAgN0ADp93IECdz3CBUJKTDsNwgiiJA6CbzvCjSh7PsACFaHgHA4EICBCSiCB4BHKANDgQIcH7OBcWiY9OFMCRp0Qj1CAAcTseIGj+AFN0YMQiVJTgeEHZ9knwWgMBYeIz3EaSNMM4zyGSHgLFfd98E-bkfzUmSGmA9ytLA-TiXsk8oPU2SjJPeCZznZTvLsrT0MkKjyRo4BCIgYigoaSLn2YlKcLw9K6NsiKHOYiwgA), which works in 4.10.2 but does not in 3.12.0.

Only [documented breaking changes between 3 and 4](https://github.com/sindresorhus/type-fest/releases/tag/v4.0.0) are:

- require Node 16+ ✅ 
- require TS 5.1+ ✅ 
- removing a deprecated function that we don't use anywhere